CDC-Data-Reconciliation-Backend/compare.py

```diff
@@ -5,14 +5,15 @@
 
 
 class CaseResult:
-    def __init__(self, caseID, eventCode, eventName, MMWRYear, MMWRWeek, reason, reasonID) -> None:
+    def __init__(self, caseID, eventCode, eventName, MMWRYear, MMWRWeek, reason, reasonID, caseClass) -> None:
```

Comment:
A bit of a nit: but I think we should change caseClass to caseClassStatus to match the fields in the csv files and query.